client/webserver/site/src/js/markets.js

```diff
@@ -98,13 +105,24 @@ export default class MarketsPage extends BasePage {
     this.ordersSortDirection = 1
     // store original title so we can re-append it when updating market value.
     this.ogTitle = document.title
-    const reporters = {
-      click: p => { this.reportClick(p) },
-      volume: d => { this.reportVolume(d) },
-      mouse: d => { this.reportMousePosition(d) },
-      zoom: z => { this.reportZoom(z) }
+
+    const depthReporters = {
+      click: p => { this.reportDepthClick(p) },
+      volume: d => { this.reportDepthVolume(d) },
+      mouse: d => { this.reportDepthMouse(d) },
+      zoom: z => { this.reportDepthZoom(z) }
+    }
+    this.depthChart = new DepthChart(page.marketChart, depthReporters, State.fetch(depthZoomKey))
+
+    const candleReporters = {
+      mouse: c => { this.reportMouseCandle(c) }
```

Comment:
When I go from depth -> candles -> depth, the mouse position seems to be reported incorrectly.  The vertical line as well as the clicked locations are shifted to the right.

https://user-images.githubusercontent.com/9373513/133164351-fba24021-24cc-40c1-9654-7c3b46cb41eb.mp4